Contributing.md

```diff
@@ -29,7 +26,7 @@ Following this guide and formatting your code as detailed will likely get your p
 This project uses clang-format (stable branch) to check for common style issues. In case of conflicts between this guide and clang-format rules, the latter should be followed instead of this guide.
 
 
-### Checking and fixing formatting issues
+## <a name="intro-formatting-issues"></a>Checking and fixing formatting issues
```

Comment:
I'm following the general formatting conventions of the ISO C++ guidelines so that it's possible to link to it and still be consistent across documents.